docs/line.js.html

```diff
@@ -580,20 +539,54 @@ <h1 class="page-title">Source: line.js</h1>
 
         /**
          * Parses dates and values into JS Date objects and numbers
-         * @param  {obj} data           Raw data
-         * @param  {obj} dataByDate     Raw data ordered by date
-         * @return {obj}            Parsed data with dates
+         * @param  {obj} dataByTopic    Raw data grouped by topic
+         * @return {obj}                Parsed data with dataByTopic and dataByDate
          */
-        function cleanData({data, dataByDate}) {
-            data.forEach(function(kv) {
-                kv.Data.forEach(function(d) {
-                    d.date = new Date(d.fullDate);
+        function cleanData({dataByTopic, dataByDate}) {
```

Comment:
it would be interesting if we could think of a dynamic way to index the dataset by the key that you'd like, instead of having to pass it in. Not quite sure how it would play out yet, but it would be nice!